Week2/LaurAlejandraC/spring-rest/src/main/java/bootcamp/springrest/dateRange/controller/DateRangeController.java

```diff
@@ -0,0 +1,33 @@
+package bootcamp.springrest.dateRange.controller;
+
+import bootcamp.springrest.dateRange.domain.DateRange;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@RestController
+@RequestMapping("/date")
```

Comment:
The resources names usually should be on plural, like /dates, due to they are collections of elements.